poetry.lock

```diff
@@ -0,0 +1,3920 @@
+# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
```

Comment:
Remove this file

Reply:
This `.lock` file make sure the dependencies has the same versions in any environment. Keeping this file may be a good idea, but it also has some problems. See https://python-poetry.org/docs/basic-usage/#as-a-library-developer. How do you think?

Reply:
I think we don't need `.lock` for now since we will update dependencies quite frequently.